tests/layer_tests/mo_python_api_tests/test_mo_convert_pytorch.py

```diff
@@ -171,7 +175,7 @@ def create_pytorch_nn_module_sample_input_int32(tmp_dir):
     ref_model = make_ref_pt_model_one_input(shape, dtype=numpy.int32)
 
     return pt_model, ref_model, {'example_input': sample_input,
-                                 'input_shape': shape, "use_legacy_frontend": True}
+                                 'input': (shape, np.int32)}
```

Comment:
do we have any OOB test case for PT models? Will it work now?

Reply:
We need at least types for such a simple model like tested here

Reply:
I added OOB test, no `input` and no `example_input` no other parameters.